src/store/reducers/appReducer.js

```diff
@@ -67,7 +96,11 @@ const initialState = {
   deviceGroups: {},
   activeDeviceGroupId: undefined,
   theme: 'dark',
-  version: '0.0.1' // TODO: Version should be requested from the service
+  version: undefined,
```

Comment:
null instead of undefined

Reply:
Code guide says "Use undefined, not null"